libs/core/src/components/pds-text/docs/pds-text.mdx

```diff
@@ -61,39 +61,42 @@ Text can be aligned to the start, center, end, or justified.
 
 ### Color
 
-Text color can align to our common variant/sentiment values.
+Text color can align to our common variant/sentiment values, Pine tokens, or custom colors.
 
 <DocCanvas client:only
   mdxSource={{
     react: `
-      <PdsText style="background-color: #33; padding-inline: 8px" color="primary" tag="p">Zebstrika</PdsText>
-      <PdsText color="secondary" tag="p">Pangoro</PdsText>
-      <PdsText color="neutral" tag="p">Urshifu</PdsText>
-      <PdsText color="accent" tag="p">Sableye</PdsText>
-      <PdsText color="danger" tag="p">Arcanine</PdsText>
-      <PdsText color="info" tag="p">Garchomp</PdsText>
-      <PdsText color="success" tag="p">Sceptile</PdsText>
-      <PdsText color="warning" tag="p">Pikachu</PdsText>
+      <PdsText style="background-color: #33; padding-inline: 8px" color="primary" tag="h5">Zebstrika</PdsText>
```

Comment:
I think the inline styling syntax here may be backwards with the web component. Meaning react uses the camelCase.

Reply:
Fixed